sys/dev/axgbe/xgbe-phy-v2.c

```diff
@@ -2890,12 +2890,10 @@ xgbe_phy_link_status(struct xgbe_prv_data *pdata, int *an_restart)
 		phy_data->rrc_count = 0;
 		if (pdata->link_workaround) {
 			ret = xgbe_phy_reset(pdata);
-			if (ret) {
+			if (ret)
 				axgbe_error("Error resetting phy\n");
-			}
-		} else {
+		} else
```

Comment:
dangling else, what could possibly go wrong... some upstream style requests are just fuel to the fire :D

Reply:
I know right? Not my decision :D

Reply:
Yeah, no worries. Let's go with the flow.